sensing/autoware_pointcloud_preprocessor/src/distortion_corrector/distortion_corrector_node.cpp

```diff
@@ -92,10 +93,32 @@ void DistortionCorrectorComponent::onPointCloud(PointCloud2::UniquePtr pointclou
     return;
   }
 
-  distortion_corrector_->setPointCloudTransform(base_frame_, pointcloud_msg->header.frame_id);
-
+  distortion_corrector_->set_pointcloud_transform(base_frame_, pointcloud_msg->header.frame_id);
   distortion_corrector_->initialize();
-  distortion_corrector_->undistortPointCloud(use_imu_, *pointcloud_msg);
+
+  if (update_azimuth_and_distance_ && !angle_conversion_opt_.has_value()) {
```

Comment:
I kind of feel this block of code belongs inside the `distortion_corrector.cpp`.
For example inside of initialize.
This is just my personal opinion so feel free to ignore it (close this conversation in that case)